src/renderer/components/ft-playlist-add-video-prompt/ft-playlist-add-video-prompt.vue

```diff
@@ -48,14 +57,19 @@
       <ft-flex-box>
         <div
           v-for="(playlist, index) in activePlaylists"
-          :key="playlist._id"
+          :key="`${playlist._id}-${playlistDisabled(playlist._id)}`"
           class="playlist-selector-container"
+          :class="{
+            disabled: playlistDisabled(playlist._id),
+          }"
```

Comment:
```suggestion
          }"
          :aria-disabled="playlistDisabled(playlist._id)"
```

There are still ways to focus non-focusable elements with screenreader tech, so best to just cover our bases semantically